docs/product-user-guide/src/main/asciidoc/decision-tables-attributes-ref.adoc

```diff
@@ -0,0 +1,65 @@
+[id='decision-tables-attributes-ref']
+= Additional rule attributes for RuleSet or RuleTable definitions
+
+The `RuleSet` and `RuleTable` areas also support labels and values for other rule attributes, such as `PRIORITY` or `NO-LOOP`. Rule attributes specified in a `RuleSet` area will affect all rule assets in the same package (not only in the spreadsheet). Rule attributes specified in a `RuleTable` area will affect only the rules in that rule table. You can use each rule attribute only once in a `RuleSet` area and once in a `RuleTable` area. If the same attribute is used in both `RuleSet` and `RuleTable` areas within the spreadsheet, then `RuleTable` takes priority and the attribute in the `RuleSet` area is overridden.
+
+The following table lists the supported labels (column headers) and values for additional `RuleSet` or `RuleTable` definitions. For column headers, you can use either the given labels or any custom labels that begin with the letters listed in the table.
+
+.Additional rule attributes for `RuleSet` or `RuleTable` definitions
+[cols="30%,15%,55%", options="header"]
+|===
+|Label
+|Or custom label that begins with
+|Value
+
+|`PRIORITY`
+|P
+|An integer defining the `salience` value for the rule. Overridden by the `Sequential` flag.
```

Comment:
Probably not an issue, but just to be sure. Does the product actually use the term "salience"? My only concern would be how translatable it would be.

Reply:
Yes. It's one of the available rule attributes.